app/components/aside_component/aside_component.html.erb

```diff
@@ -11,7 +11,7 @@
     </div>
   <% end %>
   <% if @title.present? %>
-    <h2 class="<%= @use_button? 'govuk-heading-m' : 'govuk-heading-s' %> aside-component__heading">
+    <h2 class="<%= 'govuk-heading-s' %> aside-component__heading">
```

Comment:
This doesn't need to be interpolated, you can just write it as a bare class name now.

Reply:
Resolved thank you. I overlooked that one somehow